c3po/core.py

```diff
@@ -52,8 +54,16 @@ def get_feed():
     response = REQ_SESSION.get(request_url, params=PAYLOAD)
     if response.status_code == 400:
         refresh_short_token()
-    print(response.json())
-    return response.json()
+    dict = response.json()
+
+    while ('paging' in dict):
+        x = dict['paging']['next']
+        request_url = x + LTTK_GROUP_ID + '/feed'
```

Comment:
Pretty sure you don't need to do this. `x`is already the url for the next batch of posts in the feed. Meaning it already contains the `/feed` edge.

Reply:
Also, use better variable names. Variable names must always be self-explanatory. `x` tells me nothing, but `request_url` does. It's okay to reuse variables (as long as you're doing it carefully). It should be fine to reuse `request_url` here.